UserListScreen.tsx

```diff
@@ -133,6 +154,29 @@ export default class UserList extends Component {
           </ApolloProvider>
         );
     }
+
+    renderFooter() {
+      return (
+        <View style={styles.footer}>
+          <TouchableOpacity
+            activeOpacity={0.9}
+            onPress={() => {
+              if (this.hasNextPage) {
+                this.setState({offset: this.state.offset + VISIBLE_USERS_LIMIT});
+              } else {
+                alert("Todos os usuários foram carregados.");
+              }
+            }}
+            style={styles.loadBtn}>
+              {this.fetchingFromServer ? (
```

Comment:
Eu não vi essa property sendo setada pra `true` 🤔, esse load some depois da primeira vez que aparece?

Reply:
Dentro do método `fillUpList()` a variável é setada de acordo com o `loading` vindo do servidor `this.fetchingFromServer = loading;`. Nesse caso, a variável volta pra `false` no momento em que recebe o dado novamente do servidor.